VDF.Core/FFTools/FFProbeJsonReader.cs

```diff
@@ -18,114 +18,212 @@
 using System.Collections.Generic;
 using System.Text.Json;
 using VDF.Core.Utils;
+using System.Text.Json.Serialization;
+
+using DSO = System.Collections.Generic.Dictionary<string, object>; 
+using LO  = System.Collections.Generic.List <object>; 
```

Comment:
Don't use abbreviations, remove them please

Reply:
I was so happy when I found a way to shorten this because `typedef` doesn't seem to exist in C#.
Do you generally not define your own types for such purposes in C#, or is only the variant with using not the best way?

Reply:
I don't use abbreviations. If you see them in code, especially in code you haven't written yourself. You have to look up what they mean. And the more abbreviations and the more complex the code, the harder it can be read.